### ISSUE TITLE
Exception thrown while Starting an API-M profile with -Dprofile=api-store

### ISSUE THREAD

**Issue by sarubi** [wso2/product-apim#3287]
Description:
While starting the server following exception were throwing,

APIManagerComponent Exception occurred while creating WSO2 Event Adapter. Request Blocking may not work properly org.wso2.carbon.event.output.adapter.core.exception.OutputEventAdapterException: Output Event Adapter not created as no adapter factory is registered with type wso2event

Suggested Labels:
APIM 2.5.0-SNAPSHOTSNAPSHOT, BUG

Suggested Assignees:
N/A

Affected Product Version:
N/A

OS, DB, other environments details and versions:
N/A

Steps to reproduce:
I have followed steps mentioned here.
After successfully setting up the configurations mentioned in step-3, I got an error in executing command mentioned in step-4. (sh <API-M_HOME>/bin/wso2server.sh -Dprofile=api-store)

Related Issues:
N/A

**Comment by Krishanx92** [comment]
@sarubi Did you get log warn message or the exception?
WARN - APIManagerComponent Exception occurred while creating WSO2 Event Adapter. Request Blocking may not work properly.

**Comment by sarubi** [comment]
@Krishanx92 
Yes I got log WARN message. Same issue occurred while starting the api-key-manager  profile as well.
Also I noticed that, it got resolved after I set "EnableAdvanceThrottling" value to false in configuration for both scenarios.

**Comment by chirangaalwis** [comment]
@Krishanx92 came across the same issue as @sarubi has mentioned earlier.

I experienced that this gets fixed with the solution provided by @Krishanx92.

**Comment by sarubi** [comment]
Rather than the store, key-manager profiles, I got the same issue while starting the gateway worker node profile as well. 
Also, it gets solved by the solution provided by @Krishanx92 (disable the <PolicyDeployer>).

**Comment by ClimberBear** [comment]
I have the same issue starting with -Dprofile=gateway-manager in version 2.1.0-update14.
But, where have I to modify (and how) PolicyDeployer?
We have this only in `api-manager.xml` as
```
    <PolicyDeployer>
      <ServiceURL>https://apimanager.example.com:9443/services/</ServiceURL>
      <Username>${admin.username}</Username>
      <Password>${admin.password}</Password>
    </PolicyDeployer>
```
How change it to "false"?

**Comment by sarubi** [comment]
@ClimberBear, Yes I noticed that one, that's why I confused, I tried it by removing "PolicyDeployer" also It didn't work. 
@Krishanx92, Could you please check it?

**Comment by Krishanx92** [comment]
@sarubi @ClimberBear PolicyDeployer disables property coming after apim-2.2.0 and onwards.
In APIM-2.1.0 updates you can resolve by setting "EnableAdvanceThrottling" value to false.

**Comment by tgtshanika** [comment]
This is fixed in APIM 2.6.0

**Comment by polikmage** [comment]
Hi, it is not fixed in 2.6.
